internal/worker.go

```diff
@@ -266,6 +266,27 @@ type (
 		// Optional: Host is just string on the machine running the client
 		// default: empty string
 		Host string
+
+		// Optional: See WorkerBugPorts for more details
+		WorkerBugPorts WorkerBugPorts
+	}
+
+	// WorkerBugPorts allows opt-in enabling of older, possibly buggy behavior, primarily intended to allow temporarily
+	// emulating old behavior until a fix is deployed.
+	// By default, bugs (especially rarely-occurring ones) are fixed and all users are opted into the new behavior.
+	// Back-ported buggy behavior *may* be available via these flags.
+	//
+	// Bugports are always deprecated and may be removed in future versions.
+	// Generally speaking they will *likely* remain in place for one minor version, and then they may be removed to
+	// allow cleaning up the additional code complexity that they cause.
```

Comment:
If you add `Deprecated: ...`, does vscode hint you about it? https://github.com/golang/go/issues/40447 

personally I've had that succeed and fail somewhat randomly through the years, but goland has been consistent about showing it.  and in principle it should be supported.

Reply:
I got mixed signals from codebase so left the original comments from [child-workflow-options bugport](https://github.com/taylanisikdemir/cadence-client/blob/a73fe0e8ea2f180446d828ba5786a30b0668123f/internal/workflow.go#L504). Let's use with `DEPRECATED:` format going forward as the standard. Just tried and it hints me on VSCode when the deprecated field is referred on another package.